karma.conf.js

```diff
@@ -61,40 +62,16 @@ var localLaunchers = {
 	}
 };
 
-const babelOptions = (options = {}) => {
```

Comment:
If my understanding is correct, these separate options are needed to accomplish two things:
1. Selectively apply `transform-react-jsx-source` plugin for tests that need it.
2. Add istanbul plugin for code coverage

For (1) I've moved that special override to the babel.config.js file using the override section to selectively add that that plugin only the the relevant test files using the same regex specified in the webpack loader. All the tests still pass, so I'm assuming this works as expected.

For (2) I just inlined that config in the webpack loader since that is the only place it needs to be specified. I'm uncertain if there is a way to do this in the babel.config.js and only have it applied to source files during test. If so, it might be nice to just have one babel config, but this works for now.

Reply:
yup, I only did that to enable the plugin for certain files and didn't know any better way to do it :+1: Glad there is a better one :100: